content/v3/auth.md

```diff
@@ -33,7 +33,7 @@ would authenticate you if you replace `<username>` with your GitHub username.
 (cURL will prompt you to enter the password.)
 
 ``` command-line
-$ curl -u <username> https://api.github.com/user
```

Comment:
You can replace these with `em` tags to have them render as variables. For example:

    ``` command-line
    $ curl -u <em>username</em> https://api.github.com/user
    ```